glyphs-reader/src/font.rs

```diff
@@ -2051,6 +2073,14 @@ fn preprocess_unparsed_plist(s: &str) -> Cow<str> {
 
 impl Font {
     pub fn load(glyphs_file: &path::Path) -> Result<Font, Error> {
+        let mut font = Self::load_impl(glyphs_file)?;
+        font.propagate_all_anchors();
```

Comment:
I feel like this `propagate_all_anchors ` should be optional somehow, or at least there should be a method to _only_ load the .glyphs file without the anchor propagation.. It's not our current use case in fontc, but it's conceivable that one may wish to read a .glyphs file, change some bit that's unrelated to anchors, and write it back to a file to do something else.